apis/fluentbit/v1alpha2/plugins/output/prometheus_remote_write_types.go

```diff
@@ -0,0 +1,93 @@
+package output
+
+import (
+	"fmt"
+
+	"github.com/fluent/fluent-operator/apis/fluentbit/v1alpha2/plugins"
+	"github.com/fluent/fluent-operator/apis/fluentbit/v1alpha2/plugins/params"
+	"github.com/fluent/fluent-operator/pkg/utils"
+)
+
+// +kubebuilder:object:generate:=true
+
+// An output plugin to submit Prometheus Metrics using the remote write protocol
+type PrometheusRemoteWrite struct {
+	// IP address or hostname of the target HTTP Server, default: 127.0.0.1
+	Host string `json:"host"`
+	// Basic Auth Username
+	HTTPUser *plugins.Secret `json:"httpUser,omitempty"`
+	// Basic Auth Password.
+	// Requires HTTP_user to be se
+	HTTPPasswd *plugins.Secret `json:"httpPassword,omitempty"`
+	// TCP port of the target HTTP Serveri, default:80
+	// +kubebuilder:validation:Minimum:=1
+	// +kubebuilder:validation:Maximum:=65535
+	Port *int32 `json:"port,omitempty"`
+	// Specify an HTTP Proxy. The expected format of this value is http://HOST:PORT.
+	Proxy string `json:"proxy,omitempty"`
+	//Specify an optional HTTP URI for the target web server, e.g: /something ,default: /
+	Uri string `json:"uri,omitempty"`
+	//Add a HTTP header key/value pair. Multiple headers can be set.
+	Headers map[string]string `json:"headers,omitempty"`
+	//Log the response payload within the Fluent Bit log,default: false
+	LogResponsePayload *bool `json:"logResponsePayload,omitempty"`
+	//This allows you to add custom labels to all metrics exposed through the prometheus exporter. You may have multiple of these fields
+	AddLabels []string `json:"addLabels,omitempty"`
```

Comment:
The `AddLabels` should be similar to `Headers`, allowing to add multiple kv pairs:
https://docs.fluentbit.io/manual/pipeline/outputs/prometheus-remote-write
![image](https://user-images.githubusercontent.com/18525465/185888487-16aefe01-5181-4adb-a2c0-093da4697ddb.png)

https://docs.fluentbit.io/manual/pipeline/outputs/opentelemetry
![image](https://user-images.githubusercontent.com/18525465/185888621-d2420635-a417-46b5-a65e-6ad7c7e59ee7.png)